Source/Core/Common/Crypto/ec.cpp

```diff
@@ -278,8 +278,10 @@ std::array<u8, 60> Sign(const u8* key, const u8* hash)
   return signature;
 }
 
-UNUSED static int check_ecdsa(u8* Q, u8* R, u8* S, const u8* hash)
+bool VerifySignature(const u8* public_key, const u8* signature, const u8* hash)
 {
+  const u8* R = signature;
+  const u8* S = signature + 30;
```

Comment:
Is this `+ 30` important enough to get a comment? Considering there is one parameter serving two purposes.

Then again, that number is all over this method...

Reply:
IMO we *should* replace these magic numbers with named constants, but let's leave this for another PR, as #6866 already makes a lot of changes.